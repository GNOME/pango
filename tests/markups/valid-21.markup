<span rise="2.5pt">test</span>
