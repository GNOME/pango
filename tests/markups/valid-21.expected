test


---

range 0 4
0 4 rise 2560
range 4 2147483647


---

[0:4] (null) Normal
[4:2147483647] (null) Normal
